### DISCUSSION TITLE
Uncaught TypeError: Class extends value undefined is not a constructor or null

### DISCUSSION THREAD

**Discussion by tejasnayak25** [pixiv/three-vrm#1115] [Help] [answered]
Uncaught TypeError: Class extends value undefined is not a constructor or null
    at three-vrm-core.module.js:13:35
    at three-vrm.js:12:80
    at three-vrm.js:13:3
(anonymous) @ three-vrm-core.module.js:13
(anonymous) @ three-vrm.js:12
(anonymous) @ three-vrm.js:13

**Comment by 0b5vr** [comment]
needs more descriptions. specifically your building environment.
could you provide a demo which reproduces?

**Comment by tejasnayak25** [comment]
@0b5vr I'm trying to import the downloaded file in my js file. Here is the code.
<pre>import * as THREE_VRM from 'three-vrm.js';</pre>
This line alone is giving me the above-mentioned error and when I click on the links in the error, it redirects me to the three-vrm.js file and highlights the VRM_EXPRESSION class.
Kindly help me solve this error.

**Comment by 0b5vr** [comment, marked as answer]
Are you trying to use the three-vrm on bare HTML with `import` syntax?
Are you not using any packers or bundlers?

I think you would want to use the `three-vrm.module.js` version instead.
 
Also, make sure you read the installation document of the Three.js side:
https://threejs.org/docs/#manual/en/introduction/Installation

**Reply by tejasnayak25** [reply]
Thank you so much. I tried to use an example from three-vrm examples and now it works.
